pgmpy/tests/test_models/test_BayesianModel.py

```diff
@@ -226,6 +226,11 @@ def test_remove_nodes_from(self):
         self.assertRaises(ValueError, self.G1.get_cpds, 'diff')
         self.assertRaises(ValueError, self.G1.get_cpds, 'grade')
 
+    def test__eq__(self):
+        model_copy = self.G1.copy()
+        self.assertTrue(self.G1.__eq__(model_copy))
+        self.assertFalse(self.G1.__eq__(self.G))
```

Comment:
@Vaibhavdixit02 To use the operator `==`  with custom objects we are defining the method `__eq__`. So why to write tests using  `self.assertFalse(self.G1.__eq__(self.G))`?. IMO a better way would be to use       
`self.assertFalse(self.G1 == self.G)` .